docs/how-tos.md

```diff
@@ -64,6 +65,40 @@ TodoList.propTypes = {
 }
 ```
 
+## How to connect to multiple queries at once ?
+
+The `queryConnect` HOC enables you to connect multiple queries to a component. In the following example, the `App` component needs todos items and contacts. We use `queryConnect` to retrieve them all:
+
+```jsx
+import { queryConnect } from 'cozy-client'
+import React from 'react'
+import TodoList from './TodoList'
+import Contacts from './Contacts'
+
+const App = ({ todos, contacts }) => (
+  <div>
+    {todos.fetchStatus !== 'loaded' ? <p>Loading...</p> : <TodoList items={todos.data} />}
+    {contacts.fetchStatus !== 'loaded' ? <p>Loading...</p> : <Contacts items={contact.data} />}
+  </div>
+)
+
+export default queryConnect({
+  todos: { query: client => client.all('io.cozy.todos'), as: 'todos' },
+  contacts: { query: client => client.all('io.cozy.contacts'), as: 'contact' }
+})(App)
+```
+
+`queryConnect` also allows us to make queries based on props. For example, if we want to fetch todos based on a selected contact:
```

Comment:
```suggestion
`queryConnect` also allows us to make queries based on props by passing a function instead of a plain object. For example, if we want to fetch todos based on a selected contact:
```